cmd/panacead/cmd/genoracle.go

```diff
@@ -156,6 +160,20 @@ func setOracle(cmd *cobra.Command, genState *oracletypes.GenesisState) error {
 		return fmt.Errorf("failed to get oracle endpoint: %w", err)
 	}
 
+	commMaxRateStr, err := cmd.Flags().GetString(flagOracleCommMaxRate)
+	if err != nil {
+		return fmt.Errorf("failed to get oracle commission max rate: %w", err)
+	}
+
+	commMaxRate, err := sdk.NewDecFromStr(commMaxRateStr)
+	if err != nil {
+		return fmt.Errorf("inavlid commission max rate: %w", err)
+	}
+
+	if commMaxRate.IsNegative() || commMaxRate.GT(sdk.OneDec()) {
```

Comment:
How about added code like below to line 79? :
Maybe we can replace validation.
```go
if err := oracleGenState.Validate(); err != nil {
	return err
}
```

Reply:
And we need to change `Oracle.ValidationBasic()` like `OracleRegistration.ValidateBasic()`.
```go
if m.OracleCommissionMaxRate.LT(sdk.ZeroDec()) || m.OracleCommissionMaxRate.GT(sdk.OneDec()) {
	return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "OracleCommissionMaxRate must be between 0 and 1")
}

if m.OracleCommissionRate.LT(sdk.ZeroDec()) || m.OracleCommissionRate.GT(m.OracleCommissionMaxRate) {
	return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "oracleCommissionRate must be between 0 and OracleCommissionMaxRate")
}

if m.OracleCommissionMaxChangeRate.LT(sdk.ZeroDec()) || m.OracleCommissionMaxChangeRate.GT(sdk.OneDec()) {
	return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "OracleCommissionMaxChangeRate must be between 0 and 1")
}
```

Reply:
Thanks, I fixed it.